yarn-project/acir-simulator/src/client/private_execution.ts

```diff
@@ -161,15 +161,16 @@ export class PrivateFunctionExecution {
         return Promise.resolve(ZERO_ACVM_FIELD);
       },
       enqueuePublicFunctionCall: async ([acvmContractAddress], [acvmFunctionSelector], [acvmArgsHash]) => {
+        const selector = frToAztecAddress(fromACVMField(acvmContractAddress));
         const enqueuedRequest = await this.enqueuePublicFunctionCall(
-          frToAztecAddress(fromACVMField(acvmContractAddress)),
+          selector,
```

Comment:
This takes in the targetContractAddress, why is it called selector?

In `executor` this name is used again but with different fields

Reply:
Think I copied it a round because the 32byte strings tilted me on some error codes. Will fix it.